.standard.yml

```diff
@@ -1,3 +1,6 @@
 # See: https://github.com/standardrb/standard?tab=readme-ov-file#yaml-options
+ignore:
+  - 'db/schema.rb':
+    - Rails/CreateTableWithTimestamps
```

Comment:
As soon as I touched `schema.rb`, standard started complaining about a bunch of tables. But since we auto-generate this file, the warning aren't actionable.